.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ workflows:
           bucket_prefix: dpr-artifact-store
           deploy_to_test: true # Deploy to Test Environment
           deploy_to_preprod: true # Deploy to PreProd Environment
-          cache_key: "dpr-domain-builder-build-cache"
```

Comment:
Circle CI seems to want you to add a suffix such as `-v1` instead (although you should use v2 in this instance) - as per my message in slack.

Reply:
I had to go up to v4 in the end. It looks like there were some runs that used the other version numbers.